src/include/cell_inline.h

```diff
@@ -519,8 +521,10 @@ __wt_cell_pack_ovfl(WT_SESSION_IMPL *session, WT_CELL *cell, uint8_t type, WT_TI
         /* If passed page stat information, append the row and byte counts. */
         if (ovfl_ps != NULL && __wt_process.page_stats_2022) {
```

Comment:
Here we pack after the time window so we should do the same for the address cell.